### ISSUE TITLE
VB -> C#: wrong result for empty char comparisation

### ISSUE THREAD

**Issue by KunzeAndreas** [icsharpcode/CodeConverter#892]
### VB.Net input code
```vbnet
Dim testChar As Char = Nothing
Dim testResult = testChar = ""
Assert.That(testResult, "comparing null character with empty string should be true")
```

### Erroneous output
```C#
bool testResult = Conversions.ToString(testChar) == "";    

```
### Expected output
```C#
bool testResult = testChar == char.MinValue;

```
### Details
* Product in use: VS extension
* Version in use: 9.0.0.0
* Did you see it working in a previous version, which? no

**Comment by Yozer** [comment]
@KunzeAndreas are you sure that ```testResult``` should be true? 
Running the input code on my machine results in ```testResult``` being ```False```:
https://ideone.com/usf5f6

[SharpLab.io](https://sharplab.io/#v2:DYLgbgRgNAJiDUAfAkgWwA4HsBOAXAzgAQDKAnvrgKaoCwAUAAoCuEwAlgMaEDCwAhviLd6hUYWatOJFoQCyACgCUIsaoAibVISoVuACz7ZCAQSEGjAXkIA5TLj1sAdgHMVq0Rq07cAJUr4mYFxCK299QxDCACIot0IAUUcYaQh6ROTeAXwgA===) shows that there is actually a call to Conversions.ToString generated by vb compiler.

Something like that is also converted correctly:
```vbnet
Dim testChar As Char = Nothing
Dim testResult = testChar = Microsoft.VisualBasic.ControlChars.NullChar
```

```csharp
char testChar = default;
bool testResult = testChar == ControlChars.NullChar;
```

Am I missing something?

**Comment by KunzeAndreas** [comment]
@Yozer Yes, I am sure. The test ist running in Visual Studio. Maybe the is a difference between .Net and mono?

![image](https://user-images.githubusercontent.com/7735384/168145585-9e09a139-7ac7-4729-b1ca-fa9a37d952c4.png)

in dotPeek I see this

private static void CharTest()
{
      bool condition = Operators.CompareString(Conversions.ToString(char.MinValue), "", true) == 0;
      Debug.Print(condition.ToString());
      NUnit.Framework.Assert.That(condition, "comparing null character with empty string should be true");
}

**Comment by Yozer** [comment]
Thanks @KunzeAndreas, now I see what's happening and how to reproduce it.
You need to set:
```vbnet
Option Compare Text
```

Which then will yield:
bool flag = Operators.CompareString(Conversions.ToString(c), "", **true**) == 0;

and this returns ```True```
